spectator-ext-aws/src/main/java/com/netflix/spectator/aws/SpectatorRequestMetricCollector.java

```diff
@@ -94,10 +102,29 @@ public SpectatorRequestMetricCollector(Registry registry) {
   }
 
   /**
-   * Constructs a new instance. Custom tags provided by the user will be applied to every metric.
-   * Overriding built-in tags is not allowed.
+   * Constructs a new instance using the specified handler context key and no
+   * custom tags.
+   */
+  public SpectatorRequestMetricCollector(Registry registry, HandlerContextKey<String> handlerContextKey) {
+    this(registry, Collections.emptyMap(), handlerContextKey);
+  }
+
+  /**
+   * Constructs a new instance using no handler context key and custom tags.
    */
   public SpectatorRequestMetricCollector(Registry registry, Map<String, String> customTags) {
+    this(registry, customTags, null);
```

Comment:
@brharrington Actually as I stare at this it would be really helpful for all the bean wiring, etc. to have a default context key.  What do you think of e.g. https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/handlers/HandlerContextKey.html#OPERATION_NAME?

Reply:
Here goes: https://github.com/Netflix/spectator/pull/918/commits/c089bbbc96f0257788e377b9b4305c204b13fc2c

Reply:
Operation name seems reasonable.